### DISCUSSION TITLE
Slack + GItHub integration

### DISCUSSION THREAD

**Discussion by 97nitt** [AbsaOSS/spline-spark-agent#367] [General]
Would you consider adding the [Slack + GitHub](https://github.com/marketplace/slack-github) app to the Spline repos? I would love to setup automatic notifications to a Slack channel when new releases are available.

**Comment by wajda** [comment]
Seems like it's enabled.

**Comment by 97nitt** [comment]
yep, got it working now. I must have fat fingered something last week.
